sdk/core/azure-core/inc/azure/core/http/curl/curl.hpp

```diff
@@ -26,26 +27,24 @@ namespace Azure { namespace Core { namespace Http {
     constexpr static const char* c_DefaultFailedToGetNewConnectionTemplate
         = "Fail to get a new connection for: ";
     constexpr static int c_DefaultMaxOpenNewConnectionIntentsAllowed = 10;
-  } // namespace Details
-
-  /**
-   * @brief Statefull component that controls sending an HTTP Request with libcurl thru the wire and
-   * parsing and building an HTTP RawResponse.
-   * This session supports the classic libcurl easy interface to send and receive bytes from network
-   * using callbacks.
-   * This session also supports working with the custom HTTP protocol option from libcurl to
-   * manually upload and download bytes using a network socket. This implementation is used when
-   * working with streams so customers can lazily pull data from netwok using an stream abstraction.
-   *
-   * @remarks This component is expected to be used by an HTTP Transporter to ensure that
-   * transporter to be re usuable in multiple pipelines while every call to network is unique.
-   */
-  class CurlSession : public BodyStream {
-    // connection handle. It will be taken from a pool
+// Cleaner runs every 5 min or every 1 second for unit tests
+#ifdef TESTING_BUILD
```

Comment:
This is dangerous.  Altering the timeout on all tests doesn't let us test the shipping code.

Reply:
see: https://github.com/Azure/azure-sdk-for-cpp/pull/530#discussion_r476898028